inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -1,120 +1,328 @@
+```{r describe_by_age_group_and_sex}
+descriptive(study_data_cleaned, 
+            "age_group", 
+            "sex", 
+            rowtotals = TRUE, 
+            coltotals = TRUE) %>% 
+  rename("Age" = age_group, 
+         "Female cases (n)" = Female_n, 
+         "%" = Female_prop, 
+         "Male cases (n)" = Male_n, 
+         "%" = Male_prop) %>% 
+  kable(digits = 2)
+```
 
-cols <- colnames(study_data_cleaned)
-symptom_columns <- cols[grepl(pattern = "^sym", x = cols)]
 
-survey_ciprop_foreach_binary_variable(design = suvery_design, 
-                                      variables = symptom_columns) %>%
-  knitr::kable()
+Age and gender distribution of household population covered by the survey.
+```{r age_pyramid}
+plot_age_pyramid(study_data_cleaned, 
+                 age_group = "age_group", 
+                 split_by = "sex",
+                 proportion = TRUE) + 
+  labs(y = "Proportion", x = "Age group (years)") +                    # change axis labels
+  theme(legend.position = "bottom", legend.title = element_blank())   # remove title and move legend
 ```
 
+Weighted age distribution.
+```{r age_pyramid_survey}
+# TODO: Table - Age distribution of the sample population and of the study population.
+plot_age_pyramid(survey_design_sex,
+                 age_group = "age_group",
+                 split_by = "sex", 
+                 proportion = TRUE) +
+  labs(y = "Proportion", x = "Age group (years)") +                    # change axis labels
+  theme(legend.position = "bottom", legend.title = element_blank())   # remove title and move legend
+```
 
 
 ## Mortality
 
+Reported causes of death and cause-specific mortality rates, weighted
 
 ```{r}
-suvery_ciprob(suvery_design, "died") %>%
-  knitr::kable()
+
+# you will get an extra column, for weighted counts, weighted % with CI, and mortality per 10,000 persons/day with CI
+cause_of_death <- survey_design %>% 
+  group_by(cause_of_death) %>% 
+  summarize(n = survey_total(vartype = "se", na.rm = TRUE),
+            proportion = survey_mean(vartype = "ci", na.rm = TRUE),
+            mortality = survey_ratio(as.numeric(died) * 10000, obstime, vartype = "ci")) %>% 
+  mutate(n = round(n)) %>% 
+  select(-n_se)
+
+cause_of_death %>% 
+  unite_ci("% (95% CI)", starts_with("proportion"), percent = TRUE, digits = 1) %>%
+  unite_ci("Mortality per 10,000 persons/day (95% CI)", starts_with("mortality"), percent = TRUE) %>%
+  kable()
+
+
 ```
 
+Reported causes of death and cause-specific mortality rates, by age, weighted
 ```{r}
-suvery_ciprob_stratified(suvery_design, "died", "age_group") %>%
+
+survey_design %>% 
+  tabulate_survey(cause_of_death, age_group) %>%
   kable()
+
 ```
 
+Reported causes of death and cause-specific mortality rates, by gender, weighted
+```{r}
+survey_design %>% 
+  tabulate_survey(cause_of_death, sex) %>%
+  kable()
+
+```
+
+## Morbidity
+
+```{r}
+# symptoms
```

Comment:
thing should do this as though the symptoms var was multiple choice. 
Will need to update the gen_data function for that....

Reply:
how do this multiple choice variables usually look in the data? are they somehow seperated (comma, semicolon, ...) or one column per symptom with y/n ?

Reply:
pffuuhh.... In reality it actually comes out with one per column.  (see screenshot) 
I would actually do it in in gen_data so that it comes out in one column. Then have example code to use grep and create a column for each one?

![image](https://user-images.githubusercontent.com/18347047/54312936-0c192780-45d0-11e9-8da5-5ebd51e91ed7.png)